src/OrchardCore.Modules/OrchardCore.Lists/Services/ContainerService.cs

```diff
@@ -353,28 +355,33 @@ public async Task<IEnumerable<ContentItem>> QueryContainedItemsAsync(string cont
             }
         }
 
-        private IQuery<ContentItem> ContentItemFilter(ContainedItemOptions containeditemOptions, IQuery<ContentItem> containedItems)
+        private IQuery<ContentItem> ContainedItemOptionsFilter(ContainedItemOptions containedItemOptions, IQuery<ContentItem> containedItems)
         {
-            if (!string.IsNullOrEmpty(containeditemOptions?.DisplayText))
+            if (!string.IsNullOrEmpty(containedItemOptions?.DisplayText) || containedItemOptions.Status != ContentsStatus.None)
             {
```

Comment:
Erre az `if`-re nincs szükség, mert utána mindkét feltételét külön is ellenőrzöd.

Reply:
Igaz